File: VB/GridControlCellEditingConfirmationExample/MainViewModel.vb
Imports DevExpress.Mvvm
Imports System.Collections.ObjectModel

Namespace GridControlCellEditingConfirmationExample

    Public Class MainViewModel
        Inherits ViewModelBase

        Public ReadOnly Property Source As ObservableCollection(Of Item) = New ObservableCollection(Of Item)(Item.GetData(100))
    End Class
End Namespace

File: CS/GridControlCellEditingConfirmationExample/MainViewModel.cs
using DevExpress.Mvvm;
using System.Collections.ObjectModel;

namespace GridControlCellEditingConfirmationExample {
    public class MainViewModel : ViewModelBase {
        public ObservableCollection<Item> Source { get; } = new ObservableCollection<Item>(Item.GetData(100));
    }
}
